.github/dependabot.yml

```diff
@@ -14,9 +14,19 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "chore(deps): "
+    # The version of @aws-cdk/* libraries must match those from @guardian/cdk.
+    # We'd never be able to update them here independently, so just ignore them.
+    ignore:
+      - dependency-name: "@aws-cdk"
```

Comment:
This needs to be `aws-cdk` (no `@`). I made this mistake in the other repo 😄

Reply:
Oh god, and I think I committed my typo to main in platform repo!

Reply:
Ah cool! That gives away where I copied this from 😜